src/Microsoft.Health.Fhir.Liquid.Converter/DictionaryJsonConverter.cs

```diff
@@ -41,11 +40,9 @@ private object ReadValue(JsonReader reader)
                     return ReadObject(reader);
                 case JsonToken.StartArray:
                     return ReadArray(reader);
-                case JsonToken.String:
-                    // Remove line breaks to avoid invalid line breaks in json value
-                    return Regex.Replace(reader.Value.ToString(), @"\r\n?|\n", string.Empty);
```

Comment:
Is it ok to remove this line ?

Reply:
Yes, line breaks are valid in XML but invalid in generic JSON. The logic of removing line breaks is now placed in `XmlDictionaryJsonConverter`, which is used for parsing C-CDA XML documents.